libs/barista-components/sunburst/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please remove this file as it is unused. Because you've generated the lib with the nx schematics it generated a lot more than we actually need. Please go over all boilerplate files and align them with the other components.